dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -181,42 +189,6 @@ Future<void> main() async {
         return TaskResult.failure('Building ephemeral host app Podfile.lock does not contain expected pods');
       }
 
-      section('Clean build');
-
-      await inDirectory(projectDir, () async {
-        await flutter('clean');
-      });
-
-      section('Make iOS host app editable');
-
-      await inDirectory(projectDir, () async {
-        await flutter(
-          'make-host-app-editable',
```

Comment:
This was deprecated with https://github.com/flutter/flutter/issues/54408 and is a no-op.